src/http.rs

```diff
@@ -32,8 +33,81 @@ pub fn serialised_confidence(block: u64, factor: f64) -> String {
 // 💡 HTTP part where handles the RPC Queries
 
 //service part of hyper
+#[derive(From)]
 struct Handler {
-	store: Arc<DBWithThreadMode<SingleThreaded>>,
+	store: Arc<DB>,
+}
+
+impl Handler {
```

Comment:
Isn't the http logic already under development by @kroos47 ?

Reply:
just a code refactoring is okey I guess. Not sure on merge conflicts 😅